### PR TITLE
fix #12382: enable global calculated coords / variables for all users

### LINKED ISSUE #12382: [Global Caclulated Waypoints/Variables]  make feature available for all users
### Tell us your idea!

The feature of global calculated waypoints/variables is currenty only available for developers and nightly users. This is due to its current development state. This issue is a collector issue for all things which have to be done to put the feature live for all users (beta + later production).

Following things are open for that:
- [x] #12381 
- [x] #12379 
- [x] #12378 

Apart from functionality I would expect that the GUI could use a better look than it currently has. I depend on suggestions here though, I am not very good at this. 

### What solution would you suggest?

_No response_

### Is there an alternative?

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by eddiemuc**
fix #12382: enable global calculated coords / variables for all users

### CODE REVIEW

File: main/res/values/strings.xml [hunk 1]
```diff
@@ -1507,7 +1507,7 @@
     <string name="waypoint_cache_coordinates">Cache coordinates</string>
     <string name="waypoint_calculate_coordinates">Calculate coordinates</string> <!-- Legacy: button text in coordinate input -->
     <string name="waypoint_calculated_coordinates">Calculated coordinates</string> <!-- Legacy: line in waypoint description -->
```
Comment (Lineflyer) [inline main/res/values/strings.xml:1509]: Is this one and the one above still needed?

Comment (eddiemuc) [reply main/res/values/strings.xml:1509]: @Lineflyer there is a whole lot of code in c:geo which will no longer be needed after the switch. However, I want to keep it for now and see if things stay stable for at least one release. Then I will remove all that code. 

I did the same thing with filtering, and I find this to be a usefuil pattern. To not forget about the removal I created issue #12380 a while ago. I also collect there the things which need removal.

Comment (eddiemuc) [reply main/res/values/strings.xml:1509]: That said, the two lines mentioned by you may create an "unused resource" warning. Have to check that.

Comment (eddiemuc) [reply main/res/values/strings.xml]: Removed one, kept the other (which is still used)

**Pull request merged**